src/runtime_src/core/common/xclbin_parser.cpp

```diff
@@ -850,6 +850,31 @@ get_softkernels(const axlf* top)
   return sks;
 }
 
+std::vector<aieresourcesbin_object>
+get_aieresourcesbin(const axlf* top)
+{
+  std::vector<aieresourcesbin_object> vec;
+  const axlf_section_header *pSection = nullptr;
+
+  for (pSection = ::xclbin::get_axlf_section(top, AIE_RESOURCES_BIN);
```

Comment:
```suggestion
  const axlf_section_header *pSection = nullptr;

  for (auto section = ::xclbin::get_axlf_section(top, AIE_RESOURCES_BIN);
```
please no camelCase, I see violations in this file so I understand where pSection comes from :-)